antidox/perspective.py

```diff
@@ -106,36 +110,61 @@ def contains_toxicity(perspective_response):
   return is_toxic
 
 
+def use_query(client, sql_query):
+  """make big query api request"""
+  query_job = client.query(sql_query)
+  rows = query_job.result()
+  strlst = []
+  for row in rows:
+    strlst.append(str(row.cleaned_content))
+  return strlst
+  # query_job = big_q.query("""SELECT cleaned_content FROM
+  #`wikidetox-viz.wikiconv2_201904.en_cleaned_conv` LIMIT 10""")
+  # makees bigquery api request
+
+# pylint: disable=fixme, too-many-locals
 def main(argv):
-  parser = argparse.ArgumentParser()
+  """runs dlp and perspective on content """
+  parser = argparse.ArgumentParser(description='Process some integers.')
   parser.add_argument('--input_file', help='Location of file to process')
   parser.add_argument('--api_key', help='Location of perspective api key')
+  parser.add_argument('--sql_query',)
```

Comment:
Can you add the `help` strings to `--api_query` and `--csv_file`, similar to the arguments above?

Reply:
done